test/spec/algoliasearch.helper/searchForFacetValues.js

```diff
@@ -236,3 +236,19 @@ test('client.searchForFacetValues can override the current search state', functi
 
   t.end();
 });
+
+test('an error will be thrown if the client does not contain .searchForFacetValues', function(t) {
+  var fakeClient = {
+    search() { return Promise.resolve({}); }
+  };
+  var helper = algoliasearchHelper(fakeClient, 'index', {
+    highlightPreTag: 'HIGHLIGHT>',
+    highlightPostTag: '<HIGHLIGHT',
+    query: 'iphone'
+  });
+
+  t.throws(function() {
+    helper.searchForFacetValues('facet', 'query');
+  }, /searchable/);
```

Comment:
That would make sense to expect either the actual error message, or nothing. I'm not sure `searchable` is explicit enough.